src/renderer/components/Wallet/WalletHeading/WalletHeadingPrimaryActions.vue

```diff
@@ -114,11 +117,11 @@ export default {
   watch: {
     // Never show the not-voting icon until knowing if the wallet is voting or not
     'currentWallet.address' () {
-      this.showNotVoting = false
+      this.isVoting = false
```

Comment:
In all wallets the `isVoting` field remains false, perhaps we should change it to a computed field.